sympy/stats/joint_rv_types.py

```diff
@@ -433,6 +434,76 @@ def NormalGamma(sym, mu, lamda, alpha, beta):
     """
     return multivariate_rv(NormalGammaDistribution, sym, mu, lamda, alpha, beta)
 
+#-------------------------------------------------------------------------------
+# Multivariate Normal Inverse Gamma distribution ---------------------------------------
+
+class NormalInverseGammaDistribution(JointDistribution):
+
+    _argnames = ('mu', 'lamda', 'alpha', 'beta')
+    is_Continuous = True
+
+    @staticmethod
+    def check(mu, lamda, alpha, beta):
+        _value_check(mu.is_real, "Location must be real.")
+        _value_check(lamda.is_positive, "Lambda must be positive")
```

Comment:
Should these check for `is_integer` as well (and the error message correspond to that)?